.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": ["next/core-web-vitals", "plugin:prettier/recommended"],
   "rules": {
-    "@next/next/no-img-element": 0
+    "@next/next/no-img-element": 0,
+    "@next/next/no-html-link-for-pages": 0
```

Comment:
well, generally I wouldn't disable this rule for the *whole* project, and ignore it at the line level.